connman/plugins/sailfish_ofono.c

```diff
@@ -98,6 +98,10 @@ enum connctx_handler_id {
 /* Should be less than CONNECT_TIMEOUT defined in service.c (currently 120) */
 #define ACTIVATE_TIMEOUT_SEC (60)
 #define ONLINE_CHECK_SEC  (2)
+/* In milliseconds */
+#define DELAYED_CONNECT_TIMEOUT (100)
```

Comment:
I suggest DELAYED_CONNECT_TIMEOUT_MS to make it clear which units are used

Reply:
Changed,